pytorch/inference/docker/1.12/py3/Dockerfile.graviton.cpu

```diff
@@ -59,6 +60,7 @@ RUN apt-get update \
     wget \
     unzip \
     zlib1g-dev \
+ && apt-get autoremove -y \
```

Comment:
Do you know what packages this is removing? The issue with doing this on an existing version stream (as opposed to a new version) is that we may be removing dependencies that, while not used by other applications, may be used by customers. I suggest removing this line unless we can reliably predict the behavior of packages removed - esp for an image stream that has already been released.

Reply:
Much of this is for security hardening as it would remove unused packages with possible vulnerabilities. If customers are needing a packages, we would add it to the install list explicitly (can be seen in the current install) which would not be selected during the autoremove. Will do a build and see what the autoremove is removing in this case and will comment here. Would rather keep this.

Reply:
checked both images with the option disabled and manually executed autoremove. no packages were slated for this operation as this time. I feel this is safe to keep in the images.